manifest.json

```diff
@@ -237,6 +235,37 @@
             }
           }
         ]
+      },
+      "includeTwitterMetaTags": {
+        "oneOf": [
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [false]
+              }
+            }
+          },
+          {
+            "properties": {
+              "includeTwitterMetaTags": {
+                "enum": [true]
+              },
+              "twitterUsername": {
+                "title": "admin/store.twitterUsername.title",
+                "description": "admin/store.twitterUsername.description",
+                "type": "string"
+              },
+              "twitterCard": {
+                "title": "admin/store.twitterCard.title",
+                "description": "admin/store.twitterCard.description",
+                "type": "string",
+                "default": "summary",
+                "enum": ["photo", "summary", "summary_large_image"],
+                "enumNames": ["Photo", "Summary", "Summmary Large Image"]
```

Comment:
These `enumNames` should be using message keys too since they are the texts shown to users.

Reply:
@victorhmp Since this are the names that uses for their cards, don't you think it will clearer to use this values instead of their respective translations?

Reply:
Yeep! Makes sense!